examples/aws-workspace-uc-simple/versions.tf

```diff
@@ -17,7 +17,7 @@ terraform {
 
     databricks = {
       source  = "databricks/databricks"
-      version = "=1.17.0"
+      version = "=1.24.1"
```

Comment:
Let's not fix it to a specific version, but have something like ([doc](https://developer.hashicorp.com/terraform/language/expressions/version-constraints))

```
version = ">= 1.2.0, < 2.0.0"
```

Reply:
Done!